Bottom: 6a7c40526e22009d06e1a03fc6a404e5af181ccd
Top:    e61ec61a4634e4e5348b7057377f6a79389e9737
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-11 15:14:00 +0000

lesstif: Use Coord type for line-width, not int


---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index 795b9fe..c4b314e 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -52,7 +52,7 @@ typedef struct lesstif_gc_struct
 
   Pixel color;
   const char *colorname;
-  int width;
+  Coord width;
   EndCapStyle cap;
   char xor_set;
   char erase;
